backend/models/Locations.ts

```diff
@@ -5,27 +5,35 @@ type ratingType = {
   dislikes: string[];
 };
 
+type comentType = {
+  author: string;
+  text: string;
+  likes: string[];
+  dislikes: string[];
+};
 export interface ILocation extends Document {
   locationName: string;
   coordinates: [number, number];
   photoSrc: string;
   description: string;

```

Comment:
in location controller you have written that photoSrc is an array of photo's srcs

Reply:
change name of photoSrc too

Reply:
done